guides/source/active_job_basics.md

```diff
@@ -269,6 +269,12 @@ UserMailer.welcome(@user).deliver_now
 UserMailer.welcome(@user).deliver_later
 ```
 
+You can also configure `ActionMailer::DeliveryJob` to use a specific queue
+adapter, distinct from the one set on `ActiveJob::Base`:
```

Comment:
Oh, I see what you mean - I'll remove `options.delivery_job.queue_adapter ||= :inline`. Nice catch!